Kitodo/src/main/java/org/kitodo/production/dto/ProjectDTO.java

```diff
@@ -277,4 +287,33 @@ public boolean hasProcesses() {
     public void setHasProcesses(boolean hasProcesses) {
         this.hasProcesses = hasProcesses;
     }
+
+    @Override
+    public Date getStartDate() {
+        try {
+            return Strings.isNotEmpty(this.startDate) ? new SimpleDateFormat(DATE_FORMAT).parse(this.startDate) : null;
```

Comment:
`Strings.isNotEmpty` from log4j package can maybe replaced by a method from commons.lang3.StringUtils (isBlank maybe)?

This is change should be applied on all places but I'm pointing on this only here.